mockserver-core/src/main/java/org/mockserver/httpclient/NettyHttpClient.java

```diff
@@ -101,6 +102,7 @@ && isHostNotOnNoProxyHostList(remoteAddress)) {
                 .attr(SECURE, httpRequest.isSecure() != null && httpRequest.isSecure())
                 .attr(REMOTE_SOCKET, remoteAddress)
                 .attr(RESPONSE_FUTURE, responseFuture)
+                .attr(ENFORCE_SYNCHRONOUS_COMPLETION, true)
```

Comment:
Won't this make asynchronous always on?

Reply:
I guess in this case this attribute doesn't even need to be set, as long as it is queried expecting a null.

Reply:
I just set it for completeness. I dont want to design the use of the null-check in. This makes it explicit, which I like.